src/main/java/org/broadinstitute/hellbender/tools/exome/ACNVModeller.java

```diff
@@ -121,7 +123,13 @@ public void writeACNVModeledSegmentFile(final File outFile) {
      * @param outFile   output file
      */
     public void writeModeledSegmentFile(final File outFile) {
-        SegmentUtils.writeModeledSegmentFile(outFile, segments, segmentedModel.getGenome().getSampleName());
+        SegmentUtils.writeModeledSegmentFile(outFile,
+                ACNVModeledSegmentConverter.convertACNVModeledSegmentsToModeledSegments(segments, segmentedModel.getGenome()),
+                segmentedModel.getGenome().getSampleName());
+    }
+
+    public void writeACNVModeledSegmentFileAsAllelicCapSegFile(final File outFile) {
```

Comment:
This can just be named writeAllelicCapSegFile (since the previous method is just named writeModeledSegmentFile, not writeACNVModeledSegmentFileAsModeledSegmentFile).  The verbose name in ACSModeledSegmentUtils can be kept if you like.